server/views/visits.js

```diff
@@ -0,0 +1,20 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+var visitView = require('./visit');
+
+var visitsView = {
+  render: function(visits) {
+    if (!visits.length) { visits = [visits]; }
```

Comment:
hmm. Is this any clearer? `if (!Array.isArray(visits)) { visits = [visits] }`? "if it's not an array, wrap it in an array."